package.json

```diff
@@ -14,7 +14,7 @@
     "build:pficonfont": "gulp pfIconFont",
     "build:ie11": "gulp buildIE11",
     "build:site": "npm run copy-assets && gatsby build",
-    "clean": "gatsby clean && shx -rf tmp",
+    "clean": "gatsby clean && gulp clean",
     "cli:setup": "cd build/patternfly-cli && npm install && cd ../../ && npm link ./build/patternfly-cli",
     "copy-assets": "gulp copyAssets",
     "dev": "concurrently \"npm run copy-assets\" \"gulp watchSASS\" \"gatsby develop\"",
```

Comment:
Can you remove the "shx" dependency further down in this file and run `npm install`?

Reply:
I am not seeing the dependency in this branch

Reply:
You're right! I must've removed it but forgotten to remove it from the "clean" target.